### PR TITLE
Update slider js

### PR DESCRIPTION
**Pull request by VolhaVaitulevich**
1. Added background images;
2. Updated slider with js;

### CODE REVIEW

File: pages/main/index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+window.onload = function () {
```
Comment (siarhei-papkou) [inline pages/main/index.js:1]: document.addEventListener('DOMContentLoaded', function () {

File: pages/main/index.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    if (prevButton !== null) {
+      prevButton.addEventListener("click", function (e) {
+        e.preventDefault();
+        prevSlide();
+      });
+    }
+
+    let transition = parseInt(slides.dataset.transition);
+
+    if (slides.dataset.transition === null) {
```
Comment (siarhei-papkou) [inline pages/main/index.js:31]: ternary?

**Review by siarhei-papkou — CHANGES_REQUESTED** [review]

File: pages/main/index.js (hunk 1 above)
Comment (VolhaVaitulevich) [reply pages/main/index.js]: Updated in a new commit

File: pages/main/index.js (hunk 2 above)
Comment (VolhaVaitulevich) [reply pages/main/index.js]: Updated to ternary in a new commit

File: pages/main/index.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      slidesList.style.transform = `translate3d(${
+        index * -slidesWidth
+      }px, 0, 0)`;
+      console.log(index * -slidesWidth);
+    };
+  }
+
+  const slideShows = document.querySelectorAll(SLIDER);
+
+  for (let i = 0; i < slideShows.length; i += 1) {
```
[the remark this answers is not in the document]
Comment (VolhaVaitulevich) [reply pages/main/index.js]: Changed to forEach =)

File: pages/main/index.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    let transition = parseInt(slides.dataset.transition);
+
+    if (slides.dataset.transition === null) {
+      transition = 400;
+    }
+
+    slidesList.style.transition = `${transition}ms`;
+
+    const slidesWidth = slides.clientWidth;
```
[the remark this answers is not in the document]
Comment (VolhaVaitulevich) [reply pages/main/index.js]: Added an event listener for the resize event that triggers a page reload, but I'm not sure if that's how it's supposed to work.

[the remark this answers is not in the document]
Comment (VolhaVaitulevich) [reply pages/main/index.js]: Updated one more time so that it works without reloading the page after resize

**Pull request merged**